index.html

```diff
@@ -0,0 +1,101 @@
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/back_button_test.html">Back Button Test</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/longpage.html">Long Page</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/fullscreen.html">Fullscreen</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/persistent_footer_index.html">Persistent Footer Index</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test1.html">Test 1</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test2.html">Test 2</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test3.html">Test 3</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test4.html">Test 4</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test5.html">Test 5</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test6.html">Test 6</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test7.html">Test 7</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test8.html">Test 8</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test9.html">Test 9</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/exercises/test10.html">Test 10</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module2/assignment2.html">Assignment 2</a></li>
+            </ul>
+        </div>
+        <footer data-role="footer">Created by: <a href="mailto:crbrehm@mail.mccneb.edu">Carter Brehm</a></footer>
+    </div>
+    <div data-role="page" id="module3" data-title="Carter Brehm | INFO 2439 | Module 3">
+        <header data-role="header" data-add-back-btn="true">
+            <h1>Module 3</h1>
+        </header>
+        <div role="main" class="ui-content">
+            <h2>Assignments</h2>
+            <ul data-role="listview" data-inset="true">
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module3/exercises/index.html">Hotel Page Exercises</a></li>
+                <li><a data-transition="slide" data-ajax="false" rel="external" href="module3/assignment3.html">Assignment 3</a></li>
+            </ul>
```

Comment:
Well done! I like that you've included this so easily here to make such a clean index.